chaoscenter/web/src/controllers/ActiveProjectMemberList/ActiveProjectMembers.tsx

```diff
@@ -0,0 +1,26 @@
+import { useParams } from 'react-router-dom';
+import React from 'react';
+import ActiveMembersTableView from '@views/ProjectMembers/ActiveMembersTable';
+import { ProjectMember, useGetProjectMembersQuery } from '@api/auth';
+import { generateActiveMemberTableContent } from './helper';
+
+export default function ActiveProjectMembersController(): React.ReactElement {
+  const { projectID } = useParams<{ projectID: string }>();
+  const [projectMembers, setProjectMembers] = React.useState<ProjectMember[]>([]);
+  const {
+    data,
+    isLoading,
+    refetch: getMembersRefetch
+  } = useGetProjectMembersQuery({ project_id: projectID, state: 'accepted' });
```

Comment:
Add these values in enum and consume it here.

Reply:
`state` is already having defined values, so we can't create different types.